### PR TITLE
Implement W3DDefaultDraw

### PR DESCRIPTION
**Pull request by jonwil**

### CODE REVIEW

File: src/game/client/draw/w3ddefaultdraw.cpp [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+#include "assetmgr.h"
+#include "drawable.h"
+#include "w3ddisplay.h"
+#include "w3dscene.h"
+#include "w3dshadow.h"
+
+W3DDefaultDraw::W3DDefaultDraw(Thing *thing, ModuleData const *module_data) :
+    DrawModule(thing, module_data), m_renderObject(nullptr), m_shadow(nullptr)
+{
+    if (!Get_Drawable()->Get_Template()->Get_LTA_Name().Is_Empty()) {
```
Comment (tomsons26) [inline src/game/client/draw/w3ddefaultdraw.cpp]: store drawable and template so its less of a -> chain, annoying to read...

File: src/game/client/draw/w3ddefaultdraw.cpp [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        m_renderObject = W3DDisplay::s_assetManager->Create_Render_Obj(
+            Get_Drawable()->Get_Template()->Get_LTA_Name(), Get_Drawable()->Get_Scale(), 0, nullptr, nullptr);
+        Shadow::ShadowTypeInfo info;
+        info.m_type = SHADOW_VOLUME;
+        memset(&info.m_sizeX, 0, 16);
```
Comment (tomsons26) [inline src/game/client/draw/w3ddefaultdraw.cpp]: don't do that

File: src/game/client/draw/w3ddefaultdraw.cpp (hunk 1 above)
Comment (jonwil) [reply src/game/client/draw/w3ddefaultdraw.cpp]: fixed.

File: src/game/client/draw/w3ddefaultdraw.cpp (hunk 2 above)
Comment (jonwil) [reply src/game/client/draw/w3ddefaultdraw.cpp]: fixed.

**Pull request merged**